stdlib/2.7/__builtin__.pyi

```diff
@@ -11,6 +11,7 @@ from typing import (
     MutableSet
 )
 from abc import abstractmethod, ABCMeta
+from numbers import Number as _Number
```

Comment:
In a stub you can just say `from numbers import Number` and it won't be considered exported. In fact, using the `as` form causes the checker to export `_Number`. (See PEP 484, section on Stub Files, under Additional Notes.)